src/Backend/Modules/Pages/Js/Pages.js

```diff
@@ -732,7 +732,7 @@ jsBackend.pages.extras =
 
 			html = '<div id="user-template-textarea-' + key + '" class="options">';
 			html += '<p class="p0"><label>' + label + '</label></p>';
-			html += '<textarea data-ft-label="' + label + '" cols="90" rows="15">' + text + '</textarea>';
+			html += '<textarea data-ft-label="' + label + '" cols="83" rows="15">' + text + '</textarea>';
```

Comment:
Dit lijk me niet bij deze PR te horen? Of is dit om de textarea en de editor visueel meer op elkaar te laten lijken? (even breed)